corehq/apps/data_cleaning/models.py

```diff
@@ -257,3 +259,28 @@ class BulkEditChange(models.Model):
 
     class Meta:
         ordering = ["created_on"]
+
+    def edited_value(self, case):
+        simple_transformations = {
+            EditActionType.REPLACE: lambda x: self.replace_string,
+            EditActionType.FIND_REPLACE: lambda x: x.replace(self.find_string, self.replace_string),
+            EditActionType.STRIP: str.strip,
+            EditActionType.TITLE_CASE: str.title,
+            EditActionType.UPPER_CASE: str.upper,
+            EditActionType.LOWER_CASE: str.lower,
+            EditActionType.MAKE_EMPTY: lambda x: "",
+            EditActionType.MAKE_NULL: lambda x: None,
+        }
+
+        if self.action_type in simple_transformations:
+            old_value = case.get_case_property(self.property)
+            return simple_transformations[self.action_type](old_value)
+
+        if self.action_type == EditActionType.COPY_REPLACE:
+            old_value = case.get_case_property(self.copy_from_property)
+            return old_value.replace(self.find_string, self.replace_string)
```

Comment:
this is not the intention of `COPY_REPLACE`. This should return the value of `case.get_case_property(self.copy_from_property)` WITHOUT any find and replace action. It's just replacing the value of the current `property` with the value from the `copy_from_property`

Reply:
b5dfc1fb8b9